README.md

```diff
@@ -1,2 +1,41 @@
-# to-do-list
-This simple web page will be built using webpack and served by a webpack dev server.
+# To Do List: Project 1 - List Structure
+
+![microverse icon](https://img.shields.io/badge/Microverse-blueviolet)
+
+> This simple web page will be built using webpack and served by a webpack dev server.
+
+## Built With
+
+- JavaScript ES6
+- HTML
+- CSS
+
```

Comment:
- [ ] [OPTIONAL] I suggest adding a `Get started` section with instruction on how to clone the project and use npm to start development server or deployment for exemple

Reply:
Thank you @Ismailco! I will do that when I will understand what and how to do it. :sweat_smile: